src/app/components/file-upload/file-upload.component.spec.ts

```diff
@@ -136,7 +146,13 @@ describe('FileUploadComponent', () => {
 
         const reader = window['FileReader'];
         const result = { fake: true, invalid: 'yes' };
-        window['FileReader'] = () => createMockReader(result);
+
+        // @ts-ignore
+        window['FileReader'] = class {
```

Comment:
Did the previous way not work after upgrading to TypeScript 3.5.3?

Reply:
Exactly, but I could not find out why from the TS changelog.